sky/provision/kubernetes/utils.py

```diff
@@ -16,6 +17,7 @@
 from sky import skypilot_config
 from sky.adaptors import kubernetes
 from sky.provision.kubernetes import network_utils
+from sky.provision.logging import set_logging_level
```

Comment:
According to the style guide, we would like to import modules instead of functions directly: https://google.github.io/styleguide/pyguide.html#22-imports
```suggestion
from sky.provision import logging
```